### ISSUE TITLE
[BUG] Rule no-trailing-spaces is not working

### ISSUE THREAD

**Issue by jpradelle** [yeonjuan/html-eslint#456]
If I put following HTML on playground:
```html
<!doctype html>
  <html>
    <head>
    </head>
    <body>
      <div>  </div>
    </body>
  </html>
```
With .eslintrc.json
```json
{
  "rules": {
    "@html-eslint/no-trailing-spaces": "error"
  }
}
```

Nothing is reported, but `<div>  </div>` should be in error.

**Comment by yeonjuan** [comment]
Hi @jpradelle `no-trailing-spaces` is a rule that checks for “trailing spaces”. It does not check for whitespace inside tags on the same line.

**Comment by jpradelle** [comment]
Oh ok sory, that's what is written in description indead. I don't know why understood trailing spaces arround content inside tags ... (which would not be only tailing ...)
